app/api/v2/models/orders_model.py

```diff
@@ -0,0 +1,159 @@
+from db_init import db
+import datetime
+
+
+class OrderModel:
+    """blueprint for creating a parcel delivery order"""
+
+    def __init__(self, sender_name, receiver_name, receiver_contact, weight, pickup_location, destination):
+        """function to create parcel orders """
+
+        price = 50 * weight
+        self.sender_name = sender_name,
+        self.receiver_name = receiver_name,
+        self.receiver_contact = receiver_contact,
+        self.weight = weight,
+        self.pickup_location = pickup_location,
+        self.current_location = pickup_location,
+        self.destination = destination,
+        self.price = 'Ksh' + str(price),
+        self.status = "on-transit"
+        self.time_created = datetime.datetime.now()
+
+    def create_order(self):
+        """method for creating a parcel delivery order"""
+        try:
+            db.cursor.execute(
+                """
+                INSERT INTO parcels(sender_name, receiver_name, receiver_contact, weight,pickup_location,
+                current_location,destination,price,status,time_created)
+                VALUES(%s, %s, %s, %s,%s, %s, %s, %s,%s,%s) 
+                """,
+                (self.sender_name, self.receiver_name, self.receiver_contact, self.weight,
+                 self.pickup_location, self.current_location, self.destination, self.price,
+                 self.status, self.time_created)
+            )
+            db.commit()
+            return {"message": "order created successfully"}
+        except Exception as e:
+            return {"Message": e}
```

Comment:
Kindly use a more intuitive name for the error `e`, i.e use `error` instead of `e`. Additionally, variables names and dictionary keys should start with small letters.ie. `message` instead of `Message`.

Reply:
Noted. i'll fix this